packages/react/src/components/F0Card/CardInternal.tsx

```diff
@@ -117,12 +135,27 @@ export interface CardInternalProps {
   disableOverlayLink?: boolean
 }
 
+function getImageFitClassName(imageFit: CardImageFit): string {
+  switch (imageFit) {
+    case "contain":
+      return "object-contain h-full w-full"
+    case "fit-width":
+      return "object-cover h-full w-full"
+    case "fit-height":
+      return "object-contain h-full w-auto"
+    case "scale-down":
+      return "object-scale-down h-full w-full"
+  }
+}
```

Comment:
please use a map instead of a switch, with the correct type makes imposible to forget a map, something you can not control in a switch

```suggestion
const imageFitClassMap: Record<CardImageFit, string> = {
   contain: "object-contain h-full w-full",
   "fit-width": "object-cover h-full w-full",
   "fit-height": "object-contain h-full w-auto".
   "scale-down": "object-scale-down h-full w-full"
  }
function getImageFitClassName(imageFit: CardImageFit): string {
   return imageFitClassMap[imageFit]
}
```